src/backend/_scss/application.scss

```diff
@@ -411,8 +411,18 @@ a {
 /* Track SCSS */
 
 .show-button-filter {
-  display: inline-block;
-  padding-top: 6px;
+  padding-top: 10px;
```

Comment:
When date filters are increased, it seems to be adjusting well and is in accordance with the current implementation. Please guide. Thanks!!